core/src/main/scala/com/emaginalabs/kuronometer/core/model/BuildExecutionReport.scala

```diff
@@ -0,0 +1,26 @@
+package com.emaginalabs.kuronometer.core.model
+
+case class ProjectInformation(name: Option[String], description: Option[String])
+
+case class ProjectVersion(group: Option[String], version: String)
+
+case class ProjectDefinition(info: ProjectInformation, version: ProjectVersion)
```

Comment:
I wanted `group` in project version to be optional in server side, becasue we could implement clients for another build tools wich could not provide group. In the JVM, the main build tools forces us to provide it. But I'm also thinking in tools for PHP, Node, and so on... which doesn't implements this constraint.